src/main/java/com/pjait/byt/pettracker/Pulse.java

```diff
@@ -6,32 +6,41 @@
     private double force;
 
     public Pulse(double bpm, double volume, double force){
-        this.bpm = bpm;
```

Comment:
calling setter inside the constructor, not standard practice, but works for now.